packages/dashboard/src/components/sidePanel/sections/textSettingSection/index.scss

```diff
@@ -0,0 +1,33 @@
+@import "~@cloudscape-design/design-tokens/index";
```

Comment:
Not against supporting sass, but is this required? I'm guessing you needed access to those variables?

Reply:
yes `@cloudscape-design/design-tokens` provides a tons of variables that we can use them to provide consistent looking.

Reply:
cool love that addition!